codecov.yml

```diff
@@ -4,7 +4,7 @@ coverage:
     project:
       default:
         target: 100%
-        threshold: 30%
+        threshold: 60%
```

Comment:
I change this down just for now. Let's make it be higher percentage after we get POC of these tools